tempesta_fw/http.c

```diff
@@ -3863,6 +3863,163 @@ tfw_cfgop_cleanup_whitelist_mark(TfwCfgSpec *cs)
 	memset(&tfw_wl_marks, 0, sizeof(tfw_wl_marks));
 }
 
+static int
+__cfgop_brange_hndl(TfwCfgSpec *cs, TfwCfgEntry *ce, unsigned char *a)
+{
+	unsigned int i;
+	const char *val;
+
+	if (!ce->val_n) {
+		TFW_ERR_NL("%s: At least one argument is required", cs->name);
+		return -EINVAL;
+	}
+	if (ce->attr_n) {
+		TFW_ERR_NL("Unexpected attributes\n");
+		return -EINVAL;
+	}
+
+	TFW_CFG_ENTRY_FOR_EACH_VAL(ce, i, val) {
+		unsigned long i0 = 0, i1 = 0;
+
+		if (tfw_cfg_parse_intvl(val, &i0, &i1)) {
+			TFW_ERR_NL("Cannot parse brange_uri interval: '%s'\n",
```

Comment:
This function is called for different branges, but in case of error it will always produce `...brange_uri...` log output.